### ISSUE TITLE
Expanded navigation item doesn't toggle closed on first click

### ISSUE THREAD

**Issue by aduth** [uswds/uswds#4217]
### Describe the bug

If a primary navigation item is expanded by default on page load, tapping that item won't collapse it on the first try. A second tap is required. From that point forward, tapping collapses and expands as expected.

### Steps to reproduce the bug

1. On a mobile device or with a small viewport, visit https://designsystem.digital.gov/components/overview/
2. Click on "Menu" in the top-right to show the navigation
3. Tap the expanded top-level "Components" menu item

**Expected behavior:** The Components menu items are collapsed
**Actual behavior:** Nothing happens

### Screenshots

![image](https://user-images.githubusercontent.com/1779930/120651946-4db27900-c44d-11eb-9c03-3414321ffdf4.png)

### System setup

- **USWDS version:** 2.11.12
- **Device:** iPhone
- **Operating system:** iOS 14.6
- **Browser and version:** Safari

### Additional context

In some initial debugging, it appears the problem stems from duplicate click handlers overlapping each other.

Specifically, the accordion behavior [toggles the menu closed](https://github.com/uswds/uswds/blob/4a0dfc7ec3fcba32f9999a5e0555f1796d3718bb/src/js/components/accordion.js#L25-L54), and does in-fact add the `hidden` attribute to the controlled element.

However, a second click handler in the navigation component [toggles it open once more](https://github.com/uswds/uswds/blob/4a0dfc7ec3fcba32f9999a5e0555f1796d3718bb/src/js/components/navigation.js#L100).

Ideally, I would expect only at most one click handler to be handling this event, or at least to handle it idempotently.

Possible ideas for remedy:

- Does the navigation component need to be handling this event? Is it not enough for the default accordion behavior to take effect?
- Prevent the second handling via [`stopPropagation`](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) or [`stopImmediatePropagation`](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopImmediatePropagation) in the first's event handler

**Comment by mejiaj** [comment]
Thanks for reporting this! The additional context is appreciated.

**Comment by mejiaj** [comment]
>Does the navigation component need to be handling this event? Is it not enough for the default accordion behavior to take effect?

I agree the default accordion behavior should take effect. The way it's setup on the website currently seems to follow the V1 model with a duplicate menu for mobile (`.sidenav-mobile`).

Using a similar setup currently you get this effect on desktop - [Megamenu →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-nav-test/components/preview/header--basic-mega.html) and [Basic Header →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-nav-test/components/preview/header--basic.html).

We'd need to reset the nav on Desktop but allow it to expand on mobile.

**Comment by annepetersen** [comment]
USWDS is currently understaffed, so I'm closing items that aren’t as high on the priority list right now, including this one, unfortunately.
